### ISSUE TITLE
ICCV 2017 spotlight?!

### ISSUE THREAD

**Issue by chkap** [ybsong00/CREST-Release#12]
The idea is apprently borrowed from the other one convolutional layer-based tracker CRT, but not cited.
And I can not really understand how the temporal residual learning in the proposed framework can be helpful for tracking. It is unreasonable! 

Cant believe this is ICCV 2017 spotlight.

**Comment by lawpdas** [comment]
Yes, I noticed that some details are similar to CRT when I first read this paper. 
...
